test/unit/views/components/gds/TimeoutDialogSpec.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.components.gds
+
+import base.{Injector, OverridableInjector}
+import com.typesafe.config.ConfigFactory
+import config.TimeoutDialogConfig
+import forms.Choice
+import models.SignedInUser
+import org.scalatestplus.mockito.MockitoSugar
+import play.api.data.Form
+import play.api.inject.bind
+import play.api.Configuration
+import play.api.test.FakeRequest
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+import views.ViewSpec
+import views.html.choice_page
+import models.requests.AuthenticatedRequest
+import testdata.CommonTestData.validEori
+import uk.gov.hmrc.auth.core.Enrolments
+
+import scala.collection.JavaConverters.asScalaIteratorConverter
+
+class TimeoutDialogSpec extends ViewSpec with MockitoSugar with Injector {
```

Comment:
Not a big deal but I think that if you replace `ViewSpec` with `ControllerLayerSpec` you already have `getRequest()` and `user` I have used in
```
AuthenticatedRequest(getRequest(), user)
```
But... up to you. I'm going to approve it. If you push again I'll re-approve it.